mappings/net/minecraft/item/ItemStack.mapping

```diff
@@ -2,132 +2,186 @@ CLASS bcb net/minecraft/item/ItemStack
 	FIELD a EMPTY Lbcb;
 	FIELD b MODIFIER_FORMAT Ljava/text/DecimalFormat;
 	FIELD c LOGGER Lorg/apache/logging/log4j/Logger;
-	FIELD d amount I
-	FIELD e updateCooldown I
+	FIELD d count I
+	FIELD e cooldown I
 	FIELD f item Lbbw;
 	FIELD g tag Lib;
 	FIELD h empty Z
-	FIELD i holdingItemFrame Latf;
-	FIELD j lastCheckedCanHarvestBlock Lbvo;
-	FIELD k lastCheckedCanHarvestResult Z
-	FIELD l lastCheckedCanPlaceBlock Lbvo;
-	FIELD m lastCheckedCanPlaceResult Z
+	FIELD i frame Latf;
```

Comment:
This should be at least `holdingFrame`. `frame` doesn't give any more information than the field type.

Reply:
The field is named relative to the verbosity of the encapsulation, which is descriptive enough within context imo. The methods would be called as such:
```java
itemStack.isInFrame();
itemStack.getFrame();
itemStack.setFrame(...);
```

Reply:
Dunno, I still think it would not hurt to be a little more verbose here.

Reply:
Resolved until further notice through discussion on Discord
> InsomniaKitten: how else would an item's frame be interpreted

> Pyrofab: Fair